flink-table/flink-table-runtime-blink/src/main/java/org/apache/flink/table/filesystem/FileSystemOptions.java

```diff
@@ -132,12 +133,12 @@
 	public static final ConfigOption<String> SINK_PARTITION_COMMIT_TRIGGER =
 			key("sink.partition-commit.trigger")
 					.stringType()
-					.defaultValue("partition-time")
+					.defaultValue("process-time")
 					.withDescription("Trigger type for partition commit:" +
```

Comment:
I find this config a little bit weird.

`process-time` means "**process-time** > partition dir create time + delay"

`partition-time` means "watermark > **partition-time** + delay"

Seems the two options configure different parts of the inequation. What if user wants to compare watermark with partition dir create time, or compare process-time with time extracted from partition values?

Reply:
```
'process-time': use processing time, if 'current time' > 'partition creation time' + 'delay', will commit the partition.
'partition-time': extract time from partition, if 'current event time' > 'time from partition values' + 'delay', will commit the partition.
```
`time from partition values` is a kind of event time.
`partition creation time` is from processing time.

Reply:
Maybe not all combinations make sense. But my point is these two options are counter-intuitive therefore hard to understand. `partition-time` specifies how to extract time attribute for a partition, while `process-time` specifies how to use the time attribute extracted from a partition. So we're using one config to specify different things. Perhaps the two options should be either `partition-dir-time` vs `partition-value-time`, or `process-time` vs `event-time`?

Besides, I think we should be consistent with terminologies in our docs. Is 'current time' the same thing as 'process time'? If so, let's stick to 'process time' because that's what we've been using in [other documentations](https://ci.apache.org/projects/flink/flink-docs-stable/dev/table/streaming/time_attributes.html). If not, then we need to define what 'current time' is so that users can understand.

Reply:
After some offline discussion, I think `partition-time` is more direct. Adjust the comments further.